docs/components/camera/fake-micro-server.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Configure a Fake Camera (`viam-micro-server`)"
+linkTitle: "fake"
+weight: 20
+type: "docs"
+description: Configure a camera to use for testing."
+images: ["/icons/components/camera.svg"]
+tags: ["camera", "components"]
+aliases:
+  - "/components/camera/fake/"
+component_description: "A camera model for testing."
+micrordk_component: true
+toc_hide: true
+# SMEs: Matt Perez, micro-RDK team
+---
+
+A `fake` camera in the micro-RDK is a camera model for testing.
+The camera always returns the same image, which is an image of a circle inside a diamond.
+
+## Software requirements
+
+To use this model, you must follow the [Micro-RDK Development Setup](/installation/micro-rdk-dev/), which enables you to install and activate the ESP-IDF.
+At the step [Generate a new project from the micro-RDK template](/installation/micro-rdk-dev/#generate-a-new-project-from-the-micro-rdk-template) where you create a new project with `cargo generate`, select the option to include camera module traits when prompted.
+Finish the [Micro-RDK Development Setup](/installation/micro-rdk-dev/) and return to this guide.
```

Comment:
let's put it like this because otherwise the configuration falls under the heading software requirements
```suggestion
{{< alert title="Software requirements" color="note" >}}
To use this model, you must follow the [Micro-RDK Development Setup](/installation/micro-rdk-dev/), which enables you to install and activate the ESP-IDF.
At the step [Generate a new project from the micro-RDK template](/installation/micro-rdk-dev/#generate-a-new-project-from-the-micro-rdk-template) where you create a new project with `cargo generate`, select the option to include camera module traits when prompted.
Finish the [Micro-RDK Development Setup](/installation/micro-rdk-dev/) and return to this guide.
{{< /alert >}}
```

Reply:
sounds good